extensions/sonos/src/decrease-volume.tsx

```diff
@@ -0,0 +1,24 @@
+import { LaunchType, launchCommand } from "@raycast/api";
+import { getActiveCoordinator } from "./core/sonos";
+import { SonosDevice } from "@svrooij/sonos/lib";
+import { handleCommandError } from "./core/utils";
+
+export default async function Command() {
+  let coordinator: SonosDevice | undefined;
+
+  try {
+    coordinator = await getActiveCoordinator();
+  } catch (error) {
+    await handleCommandError(error);
+    return;
+  }
+
+  if (coordinator === undefined) {
+    await launchCommand({
+      name: "set-group",
+      type: LaunchType.UserInitiated,
+    });
```

Comment:
You need to have try catch around launchCommand too (applies to all commands) 🙂

Reply:
Ah, nice catch! I've introduced `tryLaunchCommand` and use it for all those calls now.